app/frontend/src/components/FileStatus/DocumentsDetailList.tsx

```diff
@@ -409,6 +396,20 @@ export const DocumentsDetailList = ({ items, onFilesSorted}: Props) => {
 
     return (
         <div>
+            <div className={styles.buttonsContainer}>
+                <div className={`${styles.refresharea} ${styles.divSpacing}`} onClick={onRefresh} aria-label=" Refresh">
+                    <ArrowClockwise24Filled className={styles.refreshicon} />
+                    <span className={`${styles.refreshtext} ${styles.centeredText}`}>Refresh</span>
+                </div>        
+                <div className={`${styles.refresharea} ${styles.divSpacing}`} onClick={handleDeleteClick} aria-label=" Delete">
+                    <Delete24Regular className={styles.refreshicon} />
```

Comment:
It is a minor adjustment here.. but for Delete we use the 24xFilled version... for Delete and Resubmit we are using the 24xRegular icons. We should be consistent either way.